packages/@dcl/inspector/src/components/AssetPreview/AssetPreview.tsx

```diff
@@ -14,36 +17,58 @@ const WIDTH = 300
 const HEIGHT = 300
 
 export function AssetPreview({ value, resources, onScreenshot, onLoad }: Props) {
-  return (
-    <div className="AssetPreview">
-      {isGltf(value.name) ? (
-        <GltfPreview value={value} resources={resources} onScreenshot={onScreenshot} onLoad={onLoad} />
-      ) : value.name.endsWith('png') ? (
-        <PngPreview value={value} onScreenshot={onScreenshot} onLoad={onLoad} />
-      ) : (
-        <IoIosImage />
-      )}
-    </div>
-  )
+  const preview = useMemo(() => {
+    const ext = value.name.split('.').pop()
+    switch (ext) {
+      case 'gltf':
+      case 'glb':
+        return <GltfPreview value={value} resources={resources} onScreenshot={onScreenshot} onLoad={onLoad} />
+      case 'png':
+      case 'jpg':
+      case 'jpeg':
+        return <PngPreview value={value} onScreenshot={onScreenshot} onLoad={onLoad} />
+      case 'mp3':
+      case 'wav':
+      case 'ogg':
+        return <AiFillSound />
+      case 'mp4':
+        return <IoVideocamOutline />
+      default:
+        return <FaFile />
+    }
+  }, [])
+
+  return <div className="AssetPreview">{preview}</div>
 }
 
 function GltfPreview({ value, resources, onScreenshot, onLoad }: Props) {
-  const handleLoad = React.useCallback(() => {
+  const [loading, setLoading] = useState(true)
+  const handleLoad = useCallback(() => {
     onLoad?.()
     const wp = WearablePreview.createController(value.name)
-    void wp.scene.getScreenshot(WIDTH, HEIGHT).then(($) => onScreenshot($))
+    void wp.scene.getScreenshot(WIDTH, HEIGHT).then(($) => {
+      setTimeout(() => {
+        onScreenshot($)
+        setLoading(false)
+      }, 1000) // ugly hack to avoid iframe flickering...
+    })
   }, [onLoad])
 
   return (
-    <WearablePreview
-      id={value.name}
-      blob={toWearableWithBlobs(value, resources)}
-      disableAutoRotate
-      disableBackground
-      projection={PreviewProjection.ORTHOGRAPHIC}
-      camera={PreviewCamera.STATIC}
-      onLoad={handleLoad}
-    />
+    <>
+      <div className={cx('GltfPreview', { hidden: loading })}>
+        <WearablePreview
+          id={value.name}
+          blob={toWearableWithBlobs(value, resources)}
+          disableAutoRotate
+          background="#3c3c3c"
```

Comment:
mm i really dont think we should use a solid background for the image. We should have a transparent background png and set the background with css. Otherwise any change to the background or style later will look bad with a hardcoded background.